app/models/results_post_processor.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ResultsPostProcessor
```

Comment:
This is an initial implementation, and it may end up changing as we develop more of the SERP. This may be moved into a module. See Martha's comment [here](https://github.com/GSA/search-gov/pull/1197#discussion_r1179410819).